tests/gis_tests/geoapp/test_regress.py

```diff
@@ -87,3 +88,12 @@ def test_boolean_conversion(self):
         # verify values
         self.assertIs(val1, True)
         self.assertIs(val2, False)
+
+    def test_filter_on_distance_annotation(self):
```

Comment:
Try to find a logical place for this test. Ideally, we wouldn't have `test_regress.py` at all.